cvmassistants/disktool/encryptedDisk.sh

```diff
@@ -128,11 +127,10 @@ else
     umount "$MOUNT_PATH" 2>/dev/null && log_info "Unmounted $MOUNT_PATH"
 fi
 
-diskpath="/dev/$DISK" # /dev/vda
+DISK_PATH="/dev/$DISK" # /dev/vda
```

Comment:
I don't understand the uppercasing of `DISK_PATH` here.
It is a local variable and not an environment one injected by apploader.